MessageBar.js

```diff
@@ -30,46 +37,68 @@ function Message({ message }) {
 
 export default class MessageBar extends Component {
   constructor(props) {
-    super(props)
+    super(props);
     this.state = {
       isVisibleAnimValue: new Animated.Value(0),
+      isAnimatingHide: false,
       message: null,
-    }
+      config: {},
+    };
+  }
+  componentWillMount() {
+    this.panResponder = PanResponder.create({
+      onMoveShouldSetPanResponder: () => true,
+      onMoveShouldSetPanResponderCapture: () => true,
+      onPanResponderMove: (e, gesture) => {
+        if (
+          gesture.dy < -MIN_SWIPE_DISTANCE
+          && gesture.vy < -MIN_SWIPE_VELOCITY
+          && this.getConfig().closeOnSwipe
```

Comment:
Let's check this first.
If it's `false` - `gesture` params doesn't matter at all.